.circleci/config.yml

```diff
@@ -1121,4 +1121,4 @@ workflows:
           filters:
             branches:
               only:
-                - master
+                - develop
```

Comment:
Will this still build them twice per night? Once per branch.

Reply:
Very good question. I've been wondering the same. I hope not. Even if two builds should be started the duplicate one should abort since we have 'auto-cancel redundant builds' enabled.